### PR TITLE
CB-15529 addendum fix Edge Node typo

### PR DESCRIPTION
**Pull request by nkalmar**
See detailed description in the commit message.

### CODE REVIEW

**Review by bergerdenes — APPROVED** [review]

**Review by lajosrodek — CHANGES_REQUESTED** [review]
Thanks for the patch. Would you be kind to also make the changes for the 7.2.2 cluster definitions as well? There are only 3 cloud providers affected, [AWS](https://github.com/hortonworks/cloudbreak/blob/master/core/src/main/resources/defaults/clustertemplates/7.2.2/aws/edgenode.json), [Azure](https://github.com/hortonworks/cloudbreak/blob/master/core/src/main/resources/defaults/clustertemplates/7.2.2/azure/edgenode.json) and [YARN](https://github.com/hortonworks/cloudbreak/blob/master/core/src/main/resources/defaults/clustertemplates/7.2.2/yarn/edgenode.json).

I also had a note regarding 7.2.3 that might also need further consideration.

File: core/src/main/resources/application.yml [hunk 1]
```diff
@@ -414,9 +414,9 @@ cb:
           7.2.2 - Flow Management Light Duty with Apache NiFi, Apache NiFi Registry=cdp-flow-management-small;
           7.2.2 - Flow Management Heavy Duty with Apache NiFi, Apache NiFi Registry=cdp-flow-management;
           7.2.2 - Data Discovery and Exploration=cdp-dde;
-          7.2.2 COD Edge Node=cdp-cod-edge-node
+          7.2.2 - COD Edge Node=cdp-cod-edge-node
         7.2.3: >
-          7.2.3 COD Edge Node=cdp-cod-edge-node
+          7.2.3 - COD Edge Node=cdp-cod-edge-node
```
[the remark this answers is not in the document]
Comment (lajosrodek) [reply core/src/main/resources/application.yml:419]: After offline discussions with @akanto , we agreed that it makes no sense to keep this defunct entry.

@nkalmar Could you please remove the 7.2.3 section in `core/src/main/resources/application.yml` and delete `core/src/main/resources/defaults/blueprints/7.2.3/cdp-cod-edge-node.bp` as well? Please be also sure to update [ClusterTemplateTest.validateDefaultCount()](https://github.com/hortonworks/cloudbreak/blob/5e01c950f04066fe6c4f6045c23a6a4dc74b39bc/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/ClusterTemplateTest.java#L368) accordingly. (Not sure the latter is indeed needed; it is presumably only for cluster definitions.)

**Review by lajosrodek — CHANGES_REQUESTED** [review]
Thanks, changes look great! My final ask is to please squash this into a single commit, that is the usual practice for CB PRs.

**Review by lajosrodek — APPROVED** [review]
Thanks for the fixes! LGTM

**Comment by lajosrodek** [comment]
@nkalmar Could you please rebase onto top of `master`? There have been some Gradle dependency fixes that will hopefully make this PR green again. :smile:

**Comment by nkalmar** [comment]
Rebased the patch

**Comment by lajosrodek** [comment]
Various random failures this time:

spotbugs-test:
```
Execution failed for task ':client-cm:compileJava'.
> Could not resolve all files for configuration ':client-cm:compileClasspath'.
   > Could not resolve com.cloudera.api.swagger:cloudera-manager-api-swagger:7.5.3.
     Required by:
         project :client-cm
```

spotbugs-main and checkstyle-main:
```
A problem occurred configuring project ':audit-connector'.
> Could not resolve all artifacts for configuration ':audit-connector:classpath'.
   > Could not resolve com.google.protobuf:protobuf-gradle-plugin:0.8.17.
     Required by:
         project :audit-connector
```

integration-test:
```
MockSdxTests.testCustomSDXCanBeCreatedThenDeletedSuccessfully()

com.sequenceiq.it.cloudbreak.exception.TestFailException: All Exceptions that occurred during the test are logged after this message
await SdxCustomTestDto[name: mock-test-901f10af8] for desired statuses {status=RUNNING}: Cluster 'mock-test-901f10af8' entered into creation failed state. Status: '{status=PROVISIONING_FAILED}' statusReason: '{statusReason=Datalake creation failed. Cannot start provisioning, error happened during the operation: Error message: "Error with resource [STACK], error: [Key (name, workspace_id)=(mock-test-901f10af8, 2) already exists.]"}'
com.sequenceiq.it.cloudbreak.exception.TestFailException: Cluster 'mock-test-901f10af8' entered into creation failed state. Status: '{status=PROVISIONING_FAILED}' statusReason: '{statusReason=Datalake creation failed. Cannot start provisioning, error happened during the operation: Error message: "Error with resource [STACK], error: [Key (name, workspace_id)=(mock-test-901f10af8, 2) already exists.]"}'


MockSdxTests.testDefaultSDXCanBeCreatedThenDeletedSuccessfully:

await SdxInternalTestDto[name: mock-test-d0f1dcb03] for desired statuses {status=RUNNING}: Cluster 'mock-test-d0f1dcb03' entered into creation failed state. Status: '{status=PROVISIONING_FAILED}' statusReason: '{statusReason=Datalake creation failed. Cannot start provisioning, error happened during the operation: Error message: "Error with resource [STACK], error: [Key (name, workspace_id)=(mock-test-d0f1dcb03, 2) already exists.]"}'
com.sequenceiq.it.cloudbreak.exception.TestFailException: Cluster 'mock-test-d0f1dcb03' entered into creation failed state. Status: '{status=PROVISIONING_FAILED}' statusReason: '{statusReason=Datalake creation failed. Cannot start provisioning, error happened during the operation: Error message: "Error with resource [STACK], error: [Key (name, workspace_id)=(mock-test-d0f1dcb03, 2) already exists.]"}'
```

checkstyle-test:
```
Execution failed for task ':ccm-connector:extractIncludeProto'.
> Could not resolve all files for configuration ':ccm-connector:compileProtoPath'.
   > Could not resolve com.google.guava:guava:[10.+,).
     Required by:
         project :ccm-connector > com.github.rholder:guava-retrying:2.0.0
```

**Comment by lajosrodek** [comment]
Retest this please

**Review by tiborpopovics — APPROVED** [review]

**Comment by nkalmar** [comment]
Can we merge this, all green @lajosrodek ? Thanks!

**Review by pdarvasi — APPROVED** [review]

**Pull request merged**